public/src/js/utils/open-graph.js

```diff
@@ -15,7 +15,7 @@ export default function (url) {
 
         const graph = {};
         Array.prototype.forEach.call(doc.querySelectorAll('meta[property^="og:"]'), tag => {
-            graph[tag.getAttribute('property').replace(/^og\:/, '')] = tag.getAttribute('content');
+            graph[tag.getAttribute('property').replace(/^og:/, '')] = tag.getAttribute('content');
```

Comment:
Did regex ... change?

Reply:
Nope, just a linter warning about a pointless character. I think we got the warning by upgrading the linter version

Reply:
👍